auth/simple.go

```diff
@@ -70,3 +117,40 @@ func (a simpleAuth) GetUser(ctx context.Context) (string, error) {
 	}
 	return id.(string), nil
 }
+
+func (a simpleAuthN) authenticate(userID string) error {
+	if userID == "" {
+		return fmt.Errorf("Must provide header: [%s]", a.userIDHeader)
```

Comment:
I know this is not your change, but would you mind changing `Must` to `must`? Error messages should not start with upper-case letters.